src/plone/restapi/tests/test_documentation.py

```diff
@@ -809,10 +809,6 @@ def setUp(self):
         self.portal = self.layer['portal']
         self.portal_url = self.portal.absolute_url()
 
-        # Register custom UUID generator to produce stable UUIDs during tests
-        pushGlobalRegistry(getSite())
```

Comment:
@jaroel The fact that `register_static_uuid_utility()` is called multiple times isn't the issue here - the issue is that after you push a new global component registry on the stacked registry, you'll also need to **pop** it from the stack in `tearDown()` (see the [existing test case](https://github.com/plone/plone.restapi/blob/2d37bc77265cac5ea73fe9e151c4a6481dcb0b35/src/plone/restapi/tests/test_documentation.py#L167-L169)).

Also, because you don't stop `self.time_freezer` in `tearDown()`, the time freezing bleeds through to the testrunner, and that's why you get stuff like `Ran 216 tests with 0 failures and 0 errors in -17858596.730 seconds.`.